plugin/helpers.go

```diff
@@ -42,3 +46,14 @@ type Helpers interface {
 type HelpersImpl struct {
 	API API
 }
+
+func (p *HelpersImpl) ensureServerVersion(required string) error {
+	requiredVersion := semver.MustParse(required)
+	serverVersion := p.API.GetServerVersion()
+	currentVersion := semver.MustParse(serverVersion)
```

Comment:
Nit: You could combine these two lines. 0/5

Reply:
Agreed but then we need to use `currentVersion.String()` in the error message below.